core/src/main/clojure/core2/core/datalog.clj

```diff
@@ -30,7 +29,7 @@
 (s/def ::form
   (s/or :logic-var ::logic-var
         :fn-call ::fn-call
-        :value ::value))
+        :literal ::value))
 
 (s/def ::find-arg
   (s/or :logic-var ::logic-var
```

Comment:
@jms was wondering if the specs above (`fing-arg` and `form`) could be collpased or if there is something I am missing about the extra indirection?